src/Tizen.Multimedia/AudioManager/AudioDevice.cs

```diff
@@ -104,6 +107,185 @@ public bool IsRunning
                     ThrowIfError("Failed to get the running state of the device");
 
                 return isRunning;
+			}
+        }
+
+        /// <summary>
+        /// Gets the device's supported sample formats.
+        /// </summary>
+        /// <returns>An IEnumerable&lt;AudioSampleFormat&gt; that contains supported sample formats.</returns>
+        /// <remarks>
+        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction.
+        /// </remarks>
+        /// <exception cref="ArgumentException">This device is not valid or is disconnected.</exception>
+        /// <exception cref="InvalidOperationException">This device is not <see cref="AudioDeviceType.UsbAudio"/> type or is not <see cref="AudioDeviceIoDirection.Output"/> direction.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public IEnumerable<AudioSampleFormat> GetSupportedSampleFormats()
+        {
+            IntPtr formats;
+            uint numOfElems;
+
+            Interop.AudioDevice.GetSupportedSampleFormats(_id, out formats, out numOfElems).
+                ThrowIfError("Failed to get supported sample formats");
+
+            return RetrieveFormats();
+
+            IEnumerable<AudioSampleFormat> RetrieveFormats()
+            {
+                int[] formatsRes = new int[numOfElems];
+
+                Marshal.Copy(formats, formatsRes, 0, (int)numOfElems);
+                Interop.Libc.Free(formats);
+
+                IEnumerable<AudioSampleFormat> res = formatsRes.OfType<AudioSampleFormat>();
+                foreach (AudioSampleFormat f in res)
+                {
+                    Log.Debug(Tag, $"supported sample format:{f}");
+                }
+
+                return res;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the device's sample format.
+        /// </summary>
+        /// <value>The <see cref="AudioSampleFormat"/> of the device.</value>
+        /// <remarks>
+        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction.
+        /// <exception cref="ArgumentException">This device is not valid or is disconnected.</exception>
+        /// <exception cref="InvalidOperationException">This device is not <see cref="AudioDeviceType.UsbAudio"/> type or is not <see cref="AudioDeviceIoDirection.Output"/> direction.</exception>
+        /// </remarks>
+        /// <since_tizen> 5 </since_tizen>
+        public AudioSampleFormat SampleFormat
+        {
+            get
+            {
+                Interop.AudioDevice.GetSampleFormat(_id, out AudioSampleFormat format).
+                    ThrowIfError("Failed to get sample format of the device");
```

Comment:
X AVOID throwing exceptions from property getters. 
see https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property

Reply:
I'll revise it with set/get methods instead of it.